src/Network/Wai/Log/Options.hs

```diff
@@ -0,0 +1,87 @@
+module Network.Wai.Log.Options (
+-- * Options & Timing
+  Options(..)
+, ResponseTime(..)
+-- * Defaults
+, defaultOptions
+, defaultLogRequest
+, defaultLogResponse
+) where
+
+import Data.Aeson.Types (Pair)
+import Data.String.Conversions (ConvertibleStrings, StrictText, cs)
+import Data.Text (Text)
+import Data.Time.Clock (NominalDiffTime)
+import Log
+import Network.HTTP.Types.Status
+import Network.Wai
+
+-- | Logging options
+data Options = Options {
+    logLevel            :: LogLevel
+  , logRequest          :: Request -> [Pair]
+  , logSendingResponse  :: Bool
```

Comment:
I'd remove this flag if we add an ID as I suggested before

Reply:
See other comment.